README.md

```diff
@@ -145,3 +161,79 @@ Releases are aligned to Vega core [releases](https://github.com/vegaprotocol/veg
  - Run `make` followed by a full integration test run
  - Create a branch containing these changes, then a pull request into `develop`
  - Use the GitHub [New Release](https://github.com/vegaprotocol/vega-market-sim/releases/new) dialog to create a release. Tag it with the version of Vega we are matching up to (e.g. `v0.62.5`) to trigger the deployment process.
+
+
+## Windows Setup
+
+## 1. Install [WSL]
+   1. [Install Windows Subsystem for Linux (WSL) on Windows](https://learn.microsoft.com/en-us/windows/wsl/install)  
+   2. Install Ubuntu distribution inside WSL.
+  
+## 2. Install 'Make' to your WSL
```

Comment:
In markdown each entry in a numbered list can start with 1. and it will get formatted correctly. That's convenient if we ever decide to insert an intermediate step (we don't have to re-number).